src/Rings/mpoly-localizations.jl

```diff
@@ -2584,6 +2591,13 @@ function ==(f::MPolyLocalizedRingHom, g::MPolyLocalizedRingHom)
   return true
 end
 
+function Base.hash(f::MPolyLocalizedRingHom, h::UInt)
+  b = 0xe29cdc1ecb68b7a0 % UInt
+  h = hash(domain(f), h)
+  h = hash(codomain(f), h)
```

Comment:
I would add a hash of the images of the generators. Or alternatively the `restricted_map`.

Reply:
I should have re-read my code before pushing :sweat_smile: 
Done in 98c94979a6d4e4ae204aae9f6d5f695278e70fd6